src/modules/pasteplain/PastePlainModuleInterface/dllmain.cpp

```diff
@@ -0,0 +1,301 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+#include "pch.h"
+
+#include <interface/powertoy_module_interface.h>
+#include "trace.h"
+#include "Generated Files/resource.h"
+#include <common/logger/logger.h>
+#include <common/SettingsAPI/settings_objects.h>
+#include <common/utils/resources.h>
+
+#include <pastetext/PastePlainModuleInterface/PastePlainConstants.h>
```

Comment:
```suggestion
#include "PastePlainConstants.h"
```
Original path doesn't exist.